4-WebApp-your-API/4-1-MyOrg/README-use-keyvault-certificate.md

```diff
@@ -0,0 +1,77 @@
+# Web app using client certificates with Microsoft.Identity.Web
+
+There are different approaches for [using certificates with Microsoft.Identity.Web](https://github.com/AzureAD/microsoft-identity-web/wiki/Certificates).
+This document will talk about the simplest option - use of Azure KeyVault and a certificate.
+
+## Using Client certificate with KeyVault
+This sample was configured to use a client secret, but have an option to use a certificate instead. Currently this is AAD's most secure and recommended practice.
+
+### To be able to use the option you will need to make a number of changes:
```

Comment:
>the option you will need to make a number of changes: [](http://example.com/codeflow?start=22&length=53)

a certificate, please make the following changes

Reply:
done